### ISSUE TITLE
Simplify installation instructions for Windows

### ISSUE THREAD

**Issue by DC23** [carpentries-incubator/workflows-snakemake#76]
Simplify or remove the installation instructions for Windows. The CSIRO Data School uses the Git Bash shell in combination with Anaconda on Windows quite frequently. But outside of this context I think this is quite a niche usage. I think the setup instructions should be made consistent with other Python-based Carpentry material. If there is not a useful default, then I think some pointers towards WSL on Windows10 would be more up to date.

**Comment by DC23** [comment]
refers to #74

**Comment by ammaraziz** [comment]
Hi DC23, 

I'd like to take this issue on. In my opinion the main hurdle is getting linux commands running on windows. For this to be achieved there are a few choices:

1. WSL + Ubuntu 
2. Git Bash + conda 
3. Cygwin

I personally think #1 (WSL) is the best option because it gives students the most authentic Linux experience. This is important in my opinion as all (nearly all) bioinformatics happens on linux systems.  

I've tried Cygwin, it works well until it doesn't and git behaves oddly. Git + Bash is a similar situation though to be honest I haven't tested this completely. The main disadvantage with WSL + Ubuntu is the installation is more complex compared to the other two. Though all require administrator rights.

The other thing to consider is if it's worth installing Conda or go with vanilla Python. Conda is a great package manager that could really help students install bioinformatics software, avoiding the 'bullshit' with installing software. I think installing conda really opens things up to students. 

So, in summary I think the installation should be: WSL + Ubuntu + Miniconda.

Please let me know your thoughts. 

Thanks,

Ammar

**Comment by ammaraziz** [comment]
I came across discussion from another carpentrys github page that's relevant - https://github.com/carpentries/workshop-template/issues/722

An important point I missed was that WSL is not supported on all Windows version. Lots of computers are still running Win7/Win8. So a second sure-fire way to install things is needed. With that said, a second option in my opinion can be: Git Bash + miniconda

Thanks,

Ammar
